README.md

```diff
@@ -1,58 +1,9 @@
-# Architecture
+# Elemental
 
-RancherOS v2 is an immutable Linux distribution built to run Rancher and
-it's corresponding Kubernetes distributions [RKE2](https://rke2.io) 
-and [k3s](https://k3s.io). It is built using the [cOS-toolkit](https://rancher-sandbox.github.io/cos-toolkit-docs/docs/)
-and based on openSUSE. Initial node configurations is done using a
-cloud-init style approach and all further maintenance is done using
-Kubernetes operators.
+Elemental is a software stack enabling a centralized, full cloud-native cluster management solution.
 
-## Use Cases
+Cluster Node OSes are built and mainteined via container images through the [Elemental toolkit](https://rancher.github.io/elemental-toolkit/) and installed on new hosts using the [Elemental cli](https://github.com/rancher/elemental-cli).
```

Comment:
we should probably start using variables for those links and names of the stuff to have the same and be able to change them everywhere. See https://github.com/rosscdh/mkdocs-markdownextradata-plugin#features

Reply:
whhh this comment was meants for the docs, not the readme :D

Reply:
good idea, done!